app/mailers/lease_mailer.rb

```diff
@@ -5,7 +5,7 @@ def expiration_reminder
     @lease    = params[:lease]
     @property = @lease.property
     @user     = @property.user
-    @url      = user_property_lease_renew_current_lease_url(@user.id, @property.id, @lease.id, id: @lease.id)
+    @url      = user_property_lease_new_renewal_url(@user.id, @property.id, @lease.id, lease_id: @lease.id)
```

Comment:
This path was changed in a previous PR (https://github.com/happy-software/happy_house/pull/168) but forgot to update it here.